README.md

```diff
@@ -15,15 +15,12 @@ Please see the [CHANGELOG](CHANGELOG.md) for released & recent changes.
 
 ## Installation & Running
 
-We recommend installing `zulip-term` in a python virtual environment, something like the following:
-```
-[sudo] pip3 install virtualenv
-virtualenv /tmp/zt/
-. /tmp/zt/bin/activate
-pip3 install zulip-term
-```
+We recommend installing `zulip-term` in a new python virtual environment (venv); with the required python 3.4+, the following should work on most systems:
+1. `python3 -m venv zulip-terminal-venv` (creates a venv named `zulip-terminal-venv` in the current directory)
+2. `. zulip-terminal-venv/bin/activate` (activates the venv)
+3. `pip3 install zulip-term` (downloads and installs the latest zulip-terminal release from PyPI)
 
-Zulip Terminal installs as `zulip-term`, so just run:
+Zulip Terminal installs as `zulip-term`, so you can then run:
 ```
 $ zulip-term
```

Comment:
Line 23: `(zulip-terminal-venv) $ zulip-term`

Reply:
I think it serves the purpose and the change you requested is not required.